packages/react-magma-dom/src/components/TimePicker/index.tsx

```diff
@@ -326,13 +316,14 @@ export const TimePicker = React.forwardRef<HTMLInputElement, TimePickerProps>(
             id={id}
             inputStyle={{
               background: `${theme.colors.neutral08}`,
-              borderColor: `${
-                errorMessage
-                  ? theme.colors.danger
-                  : isInverse
-                  ? theme.colors.neutral08
-                  : theme.colors.neutral04
-              }`,
+              // border: '1px solid',
```

Comment:
Can we remove these commented lines?  Other than that, this looks good to me.

Reply:
Sure thing